src/sidebar/frame-sync.js

```diff
@@ -174,6 +174,16 @@ function FrameSync($rootScope, $window, Discovery, annotationUI, bridge) {
         });
       }
 
+      // if doi available as highwire or dc meta, include in search
+
+      if ( info.metadata && info.metadata.highwire && info.metadata.highwire.doi ) {
+        searchUris = searchUris.concat('doi:' + info.metadata.highwire.doi[0]);
```

Comment:
the only feedback I would have, though it's super small, is using `searchUris.push(val)` is preferred to `searchUris = searchUris.concat(val)` in this case. Less moving parts to reason about